utils/build_swift/tests/build_swift/argparse/test_types.py

```diff
@@ -197,18 +197,18 @@ def test_invalid_swift_version(self):
             swift_version_type('100.0.56.1')
 
 
-class TestShellSplitType(object):
+class TestShellSplitType(unittest.TestCase):
```

Comment:
Turns out this test class was not actually executed because it was subclassing `object`... whoops.